pkg/chain/local/local.go

```diff
@@ -53,25 +54,18 @@ func Connect() Chain {
 }
 
 func (lc *localChain) OpenKeep(keepAddress common.Address, members []common.Address) {
-	lc.handlerMutex.Lock()
-	defer lc.handlerMutex.Unlock()
-
-	lc.keeps[keepAddress] = &localKeep{
-		members: members,
-	}
-	lc.keepAddresses = append(lc.keepAddresses, keepAddress)
+	// #nosec G104 (errors unhandled)
+	// No need to handle that error for now as it is triggered only
+	// in case the keep already exists.
```

Comment:
So maybe `panic`?

Reply:
I don't think so. I'd rather ignore as actually nothing happens if the keep aleady exists.

Reply:
But it should never happen, right? I suggested `panic` as a kind of a guard condition for an invariant so that we can avoid hours of test debugging just to discover something in `local` is wrong and there are two keeps wih the same address. Maybe I don't have the full picture, though.

Reply:
With current use, it can't happen. I see your point now and I think we could add the `panic` here for now.